src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -305,6 +309,10 @@ export default class StyleConfigurator {
         // mxConstants.STYLE_HORIZONTAL is for the label
         // In BPMN, isHorizontal is for the Pool
         styleValues.set(mxConstants.STYLE_HORIZONTAL, bpmnCell.isHorizontal ? '0' : '1');
+      } else if (bpmnElement.kind === ShapeBpmnElementKind.LANE) {
```

Comment:
:warning: If you do that, you are adding support for 'vertical lanes'
- Not sure we want to do everything at the same time + missing ref to the related issue
- Missing new tests for this use case

Reply:
this is just a basis, we have doe same for pool
it does not handle complete vertical lane rendering(inside elements are not taken into account), just the shape of the lane and its label
will discuss it on call

Reply:
ok, let's put it in next PR